sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -129,8 +137,37 @@ public EditEventSteps(WebDriverHelpers webDriverHelpers, EventService eventServi
 
     When(
         "^I click on edit task icon of the first created task$",
+        () -> webDriverHelpers.clickOnWebElementBySelector(EDIT_FIRST_TASK));
+
+    When(
+        "^I click on link event group$",
+        () -> {
+          webDriverHelpers.scrollToElement(LINK_EVENT_GROUP_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(LINK_EVENT_GROUP_BUTTON);
```

Comment:
clickOnWebElementBySelector(By selector) all ready includes clickOnWebElementBySelectorAndIndex(By selector, int index) that includes scrollToElement(final Object selector), can just use one of the 2 depending if you need a wait

Reply:
I would agree with you but if i just tested it again and if i take out the scrollToElement when running with headless=false it just fails to scroll to and click on it on a lower resolution page so i would keep it just in case. With headless=true it runs fine without scrollToElement

Reply:
@VGitVlad  Richard is right, sometimes the scroll inside click method is not working and we need this approach

Reply:
Understood, Will keep an eye on this method. Resolving as is.